apps/cms/templates/meinberlin_cms/home_page.html

```diff
@@ -1,20 +1,49 @@
 {% extends "base.html" %}
-{% load wagtailcore_tags wagtailuserbar %}
+{% load wagtailcore_tags wagtailuserbar wagtailimages_tags rules %}
 
-{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}{% endblock %}
+{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}
+    {{ self.title }}{% endif %}{% endblock %}
 
 {% block header %}
     {% if self.search_description %}
-    <meta name="description" content="{{self.search_description}}" />
+        <meta name="description" content="{{ self.search_description }}"/>
     {% endif %}
 {% endblock %}
 
 {% block content %}
+    {% image self.header_image fill-1920x560-c100 as header_image %}
```

Comment:
Maybe it makes sense to use a [responsive image](https://responsiveimages.org/) here (to reduce download size on mobile).